consensus/quorum/one-node-one-vote.go

```diff
@@ -33,6 +34,25 @@ func (v *uniformVoteWeight) IsQuorumAchieved(p Phase) bool {
 	return r
 }
 
+// IsQuorumAchivedByMask ..
+func (v *uniformVoteWeight) IsQuorumAchievedByMask(mask *bls_cosi.Mask) bool {
+	threshold := v.QuorumThreshold()
+	currentTotalPower := v.computeTotalPowerByMask(mask)
```

Comment:
Just use: r := v.SignersCount(p) >= v.TwoThirdsSignersCount()   Int comparison, don't use dec, it's less error-prone.

Reply:
yeah, use countsonebit